pkgs/applications/networking/instant-messengers/qq/default.nix

```diff
@@ -0,0 +1,94 @@
+{ alsa-lib
+, cups
+, dpkg
+, fetchurl
+, gjs
+, glib
+, gtk3
+, gtk4
+, lib
+, libdrm
+, libgcrypt
+, libkrb5
+, mesa # for libgbm
+, nss
+, xorg
+, systemd
+, stdenv
+, at-spi2-core
+, autoPatchelfHook
+, wrapGAppsHook
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+let
+  mirror = "https://dldir1.qq.com/qqfile/qq/QQNT/50eed662";
+in stdenv.mkDerivation rec {
+
+  pname = "qq";
+  version = "2.0.3-543";
+
+  src = fetchurl {
+    url = "${mirror}/QQ-v${version}_x64.deb";
```

Comment:
why???

Reply:
Huh? Should I use the AppImage or the RPM instead? It is proprietary

Reply:
No, I said what this mirror string is doing here. This is not as if it is a changing target.